src/factorio_server.go

```diff
@@ -140,12 +146,27 @@ func (f *FactorioServer) Run() error {
 		ioutil.WriteFile(filepath.Join(config.FactorioConfigDir, config.SettingsFile), data, 0644)
 	}
 
-	args := []string{
-		"--bind", (f.BindIP),
-		"--port", strconv.Itoa(f.Port),
-		"--server-settings", filepath.Join(config.FactorioConfigDir, "server-settings.json"),
-		"--rcon-port", strconv.Itoa(config.FactorioRconPort),
-		"--rcon-password", config.FactorioRconPass,
+	args := []string{}
+	if config.glibcCustom == "true" {
```

Comment:
Instead of repeating the args here it would be a bit cleaner just to append the `--library-path` parameter to the args slice if it's specified.

Reply:
You are right, I'm an idiot, thanks! :)